virttest/utils_test/libvirt.py

```diff
@@ -3577,3 +3578,26 @@ def get_disk_alias(vm, source_file=None):
                 logging.info("Ignore error of source attr getting for file: %s" % e)
                 pass
     return None
+
+
+def check_cmd_expected(check_cmd, matched, is_expected):
+    """
+    check the cmd result that process run
+
+    Params check_cmd: the command need to run
+    Params matched: need to match or not match result
+    Params is_expected:
+    """
+    out = decode_to_text(process.system_output(check_cmd,
+                                               ignore_status=False,
+                                               shell=True))
+
+    if matched and not re.search(matched, out):
+        if is_expected:
+            raise exceptions.TestError(" '%s' not found in output: %s " %
+                                       (matched, out))
+
+    elif matched and re.search(matched, out):
+        if not is_expected:
+            raise exceptions.TestError(" '%s' found but expected no found in output: %s" %
```

Comment:
Better description for raise error: "%s is not expected but found in output: %s"